packages/cosmo-dycore/package.py

```diff
@@ -141,6 +138,6 @@ def cmake_args(self):
     def test(self):
       if '+build_tests' in self.spec:
           try:
-              subprocess.run(['./test_dycore.py',  str(self.spec),  self.build_directory], cwd = self.root_cmakelists_dir + '/test/tools', check=True, stderr=subprocess.STDOUT)
+              subprocess.run(['./test_dycore.py',  str(self.spec),  self.build_directory], cwd = self.root_cmakelists_dir + '/test/tools', check=True, env=None, stderr=subprocess.STDOUT)
```

Comment:
I guess `./test_dycore.py` will setup the run environment then? I still think `env=None` is too drastic, since I think it also removes the user's environment (not sure though).